package/MDAnalysis/analysis/encore/confdistmatrix.py

```diff
@@ -0,0 +1,426 @@
+# confdistmatrix.py --- Conformational distance matrix calculator
+# Copyright (C) 2014 Wouter Boomsma, Matteo Tiberti
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Distance Matrix calculation --- :mod:`MDAnalysis.analysis.ensemble.confdistmatrix`
+=====================================================================
+
+
+The module contains a base class to easily compute, using parallelization and
+shared memory, matrices of conformational distance between the structures
+stored in an Ensemble. A class to compute an RMSD matrix in such a way is also
+available.
+
+:Author: Matteo Tiberti, Wouter Boomsma, Tone Bengtsen
+:Year: 2015--2016
+:Copyright: GNU Public License v3
+:Mantainer: Matteo Tiberti <matteo.tiberti@gmail.com>, mtiberti on github
+
+.. versionadded:: 0.14.0
+
+"""
+
+from multiprocessing import Process, Array, cpu_count, Value, RawValue
+
+try:
```

Comment:
These imports need ordering so it's clear what's a global import and what is a intrapackage import, see the [style guide](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#importing-modules)